.github/workflows/build.yml

```diff
@@ -59,6 +59,10 @@ jobs:
         name: Prepare minimal package versions (MSRV only)
         run: nox -s set-minimal-package-versions
 
+      - if: inputs.rust == 'nightly' || inputs.msrv == 'MSRV'
+        name: Ignore changed error messages when using trybuild
+        run: echo "TRYBUILD=overwrite" >> "$GITHUB_ENV"
```

Comment:
This is very smart! That said, I think it would be nice if we just skipped running on msrv completely, to save some CPU cycles.

Maybe as a first line in the test we could have `if env!(PYO3_SKIP_UI_TEST) { return }` or similar? This might also solve the distro packagers problem discussed (they can probably live without UI tests).

Reply:
I am not sure if the CPU cycles are worth it, but the tests do still serve a purpose even on MSRV insofar as they check that the UI tests do fail to compile (even though with maybe a different error message compared to stable).